azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -33,9 +33,8 @@
   DISABLED(100),
   QUEUED(110),
   FAILED_SUCCEEDED(120),
-  CANCELLED(130);
-  // status is TINYINT and in H2 DB the possible values are: -128 to 127
-  // so trying to store CANCELLED in H2 fails at the moment
```

Comment:
I think it's worth preserving the information that this limitation comes from H2. Otherwise one could check MySQL and be confused as it allows larger values.